packages/@react-spectrum/picker/chromatic/Picker.chromatic.tsx

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import AlignCenter from '@spectrum-icons/workflow/AlignCenter';
+import AlignLeft from '@spectrum-icons/workflow/AlignLeft';
+import AlignRight from '@spectrum-icons/workflow/AlignRight';
+import Copy from '@spectrum-icons/workflow/Copy';
+import Cut from '@spectrum-icons/workflow/Cut';
+import {Item, Picker, Section} from '../';
+import {Meta, Story} from '@storybook/react';
+import Paste from '@spectrum-icons/workflow/Paste';
+import React from 'react';
+import {SpectrumPickerProps} from '@react-types/select';
+import {Text} from '@react-spectrum/text';
+
+const meta: Meta<SpectrumPickerProps<object>> = {
+  title: 'Picker',
+  component: Picker,
+  parameters: {
+    chromaticProvider: {colorSchemes: ['light'], locales: ['en-US'], scales: ['medium'], disableAnimations: true},
+    // chromatic needs a bit more time than disableAnimations allows
+    chromatic: {delay: 300}
+  },
+  decorators: [Story => <div style={{height: '400px'}}><Story /></div>]
+};
+
+export default meta;
+
+
+const Template = <T extends object>(): Story<SpectrumPickerProps<T>> => (args) => (
+  <Picker {...args}>
```

Comment:
It's because only one can *should* be open at a time, so it's very hard to do a bunch all in one story
They all overlap each other and at some point some code will likely exist that will make it impossible to have more than 1 open at a time anyways